### ISSUE TITLE
some questions of running runCTRecon.py 

### ISSUE THREAD

**Issue by liu123-t** [ginobilinie/medSynthesisV1#33]
Hi. I am running the code : python runCTRecon.py . some questions confused me.
 for example, "prefixModelName": is it mean a path to store trained model? 
and what is "prefixPredictedFN", can I change it?
 what is  "test_input_file_name" ?which kind image it store?

and I know"path_test indicates path for .nii.gz file", but how much datases path_test  store? all the dataset or only
 training set+validation set？
